packages/react-scripts/config/webpack.config.js

```diff
@@ -62,6 +62,9 @@ const imageInlineSizeLimit = parseInt(
 // Check if TypeScript is setup
 const useTypeScript = fs.existsSync(paths.appTsConfig);
 
+// Check disable incremental build in fork-ts-checker-webpack-plugin
+const useTypeScriptIncrementalBuild = !process.env.DISABLE_TS_INCREMENTAL_BUILD || true;
```

Comment:
if `DISABLE_TS_INCREMENTAL_BUILD  = true`，`useTypeScriptIncrementalBuild` will be `true`.